sale_order_requested_delivery/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,7 @@
+This module adds two new fields, requested_delivery_period_start and
+requested_delivery_period_end to both the sale.order and sale.order.line
+models.
```

Comment:
We should use user's label field names in module description (it has to be readable by end users / functional people, not only techies):
```suggestion
This module adds two new fields, "Customer Requested Delivery - Start" and
"Customer Requested Delivery - End" on sale orders and sale order lines.
```